extract_unique_sensors_01.py

```diff
@@ -0,0 +1,39 @@
+from urllib.request import urlopen
+
+url = ('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+try:
+    sensors_data = urlopen(url)
+except Exception:
+    filename = ('/home/lebukani/Downloads/SensorList.txt')
```

Comment:
Instead of hardcording the filepath, you can use `pathlib.Path` 

Read: 
- https://docs.python.org/3/library/pathlib.html
- Tutorial: https://realpython.com/python-pathlib/
- More Tutorial: https://pbpython.com/pathlib-intro.html